spec/javascripts/app/webauthn_spec.js

```diff
@@ -70,13 +71,11 @@ describe('WebAuthn', () => {
           },
           excludeCredentials: [
             {
-              // encodes to 'credential123'
-              id: Buffer.from([114, 183, 157, 122, 123, 98, 106, 93, 118]).buffer,
```

Comment:
Pretty sure this was always broken, since the buffer here is not actually "credential123", it's "credential12":

```
window.btoa(String.fromCharCode(...new Uint8Array([114, 183, 157, 122, 123, 98, 106, 93, 118])))
// "credential12"
```

This is clearer anyways.

This became a failure in upgrading `chai`, likely related to [release notes](https://github.com/chaijs/chai/releases/tag/4.0.0) concerning deep equality revisions.